### DISCUSSION TITLE
Sync Whatsapp folder to NC as Photo/picture folder

### DISCUSSION THREAD

**Discussion by Giuliano69** [nextcloud/all-in-one#5171] [Ideas] [answered]
Considering new Google policy about Whatsapp backup weight on disk storage, 
could be very helpful to EXTEND the "automatic backup" feature implemented for photos & screenshot, ALSO to  Whatsapp backup folders (messages and photos)

Considering Whtastapp is used by  more than 5bilions  android users (not counting Ios user...), could be a high interest feature

IF the function code is similar to the one used for   Photos folder, it could be just one more URL to add in the code...

Giuliano

**Comment by Giuliano69** [comment]
BTW... the main function (extended instant upload) is already in NC
https://github.com/nextcloud/android/pull/797

BUT  from what it seems to me, the proposed folder regard only Whatsap/Images, Whatsapp/video, .... and NOT the main Whtatsapp directory structure required for a working backup&restore.

Maybe a better initial setup could be a "root" Watshapp folder, to backup the whole thing (daily message backup, AND multimedia &documents attachment.... )

IF this is correct, the action could limit in a different preconfigured Whatsapp Folder (ready to be used as a backup solution)

What do you think ?

**Comment by szaimen** [comment, marked as answer]
Hi, please open an issue for this here: https://github.com/nextcloud/android/issues
